Soccar [Game-Server]/src/nl/soccar/gameserver/message/ChatMessage.java

```diff
@@ -2,23 +2,30 @@
 
 import nl.soccar.gameserver.message.handler.ChatMessageHandler;
 import nl.soccar.library.Player;
+import nl.soccar.library.enumeration.Privilege;
 import nl.soccar.socnet.message.Message;
 import nl.soccar.socnet.message.MessageConstants;
 import nl.soccar.socnet.message.MessageEvent;
 
 @MessageEvent(id = MessageConstants.CHAT_MESSAGE_ID, handler = ChatMessageHandler.class)
 public final class ChatMessage extends Message {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)